roles/prereqs/postgresql_connector/meta/main.yml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Cloudera, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+dependencies:
+  - role: cloudera.cluster.deployment.definition
```

Comment:
What's the deal here? Why did this ever work before?

Reply:
SSB fetches its PG connetor from a specific location, which is not the default location where the postgres connector bundled with CM is. This prereq role implements the recommendation for SSB deployment from our documentation: https://docs.cloudera.com/csa/1.5.0/installation/topics/csa-ssb-configuring-postgres.html

Reply:
Sucks but okay.